test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -76,24 +105,24 @@ namespace integration_framework {
   }
 
   IntegrationTestFramework &IntegrationTestFramework::sendTx(
-      iroha::model::Transaction tx) {
+      shared_model::proto::Transaction tx) {
     sendTx(tx, [](auto) {});
     return *this;
   }
 
   IntegrationTestFramework &IntegrationTestFramework::sendQuery(
-      const iroha::model::Query &qry) {
-    sendQuery(qry, [](const auto &) {});
+      shared_model::proto::Query qry) {
+    sendQuery(qry, [](auto) {});
```

Comment:
It does not seem right to require an explicit copy here, if it is not even used. Same applies to other lambdas.